assignments/lambda-classes.js

```diff
@@ -1 +1,189 @@
 // CODE here for your Lambda Classes
+
+// person class
+
+class Person {
+  constructor(attributes) {
+    this.name = attributes.name;
+    this.age = attributes.age;
+    this.location = attributes.location;
+    this.gender = attributes.gender;
+  }
+  speak() {
+    console.log(`Hello my name is ${this.name}, I am from ${this.location}.`);
+  }
+}
+
+// instructor class
+
+class Instructor extends Person {
+  constructor(instructorAttributes) {
+    super(instructorAttributes);
+    this.specialty = instructorAttributes.specialty;
+    this.favLanguage = instructorAttributes.favLanguage;
+    this.catchPhrase = instructorAttributes.catchPhrase;
+  }
+  demo(subject) {
+    console.log(`Today we are learning about ${subject}.`);
+  }
+  grade(x, subject) {
+    if (x.grade > 80) {
+      console.log(
+        `${x.name} receives a perfect score of ${x.grade} on ${subject}.`
+      );
+    } else {
+      console.log(
+        `${x.name} receives a weak score of ${x.grade} on ${subject}.`
+      );
+    }
+  }
+}
+
+// student class
+
+class Student extends Instructor {
```

Comment:
student should be an extend Person no Instructor, because  we dont want Student to have the attributes of Instructor, just those of Person.
```suggestion
class Student extends Person{
```